### ISSUE TITLE
Upgrade auf 3.2.0 scheitert an PHP8 Kompatibilität

### ISSUE THREAD

**Issue by jxns** [terminal42/contao-changelanguage#202]
Ich habe gerade versucht von 3.1.17 auf 3.2.0 zu updaten, erhalte jedoch folgenden Fehler:
`[1461.2MiB/9.16s] Your requirements could not be resolved to an installable set of packages.
[1461.2MiB/9.16s] 
  Problem 1
    - terminal42/contao-changelanguage[3.2.0, ..., 3.2.1] require menatwork/contao-multicolumnwizard-bundle ^3.3 -> satisfiable by menatwork/contao-multicolumnwizard-bundle[3.4.0, ..., 3.4.11].
    - menatwork/contao-multicolumnwizard-bundle[3.4.0, ..., 3.4.11] require php ^5.6 || ^7.0 -> your php version (8.0.9.0) does not satisfy that requirement.
    - Root composer.json requires terminal42/contao-changelanguage ^3.2 -> satisfiable by terminal42/contao-changelanguage[3.2.0, 3.2.1].
`

PHP8 habe ich eigentlich schon länger im Betrieb und bisher keine Probleme mit eurer Extension gehabt. Ist das Problem schon bekannt bzw. sind hier Änderungen geplant?

**Comment by fritzmg** [comment]
This extension is already compatible and unlocked for PHP 8:

https://github.com/terminal42/contao-changelanguage/blob/100330ef05982592418dd1404bec2914d0ff1105/composer.json#L29

The incompatibility comes from [menatwork/contao-multicolumnwizard-bundle](https://github.com/menatwork/contao-multicolumnwizard-bundle).

**Comment by jxns** [comment]
Yes, but the required bundle `contao-multicolumnwizard-bundle` seems not to be?

**Comment by fritzmg** [comment]
Yes, you will need to raise this issue there.
